modules/hal_nordic/nrfs/backends/nrfs_backend_ipc_service.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFS_BACKEND_IPC_SERVICE_H
+#define NRFS_BACKEND_IPC_SERVICE_H
+
+#include <stdint.h>
+#include <nrfs_common.h>
+#include <zephyr/ipc/ipc_service.h>
+#include <zephyr/kernel.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct __NRFS_PACKED ipc_data_packet_t {
```

Comment:
Can we use `__packed` here?

`struct __packed ipc_data_packet {`

Please remove ~~_t~~